config/jobs/kubernetes-sigs/cloud-provider-azure/cloud-provider-azure-config.yaml

```diff
@@ -279,7 +279,7 @@ presubmits:
         workdir: true
     spec:
       containers:
-        - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20211214-291c9e67f9-1.23
+        - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20211214-291c9e67f9-1.22
```

Comment:
The build of master branch of cloud provider azure may fail with golang v1.16.x.

Reply:
reverted this change